lib/graphql/schema.flow.js

```diff
@@ -540,9 +541,12 @@ export type Mutation = {|
   reorderCard: Card,
   /** Set the card holder representation for the customer */
   setCardHolderRepresentation: $ElementType<Scalars, 'String'>,
-  /** Categorize a transaction with an optional custom booking date for VAT or Tax categories */
+  /**
+   * DEPRECATED use `updateTransaction()` instead - Categorize a transaction with
+   * an optional custom booking date for VAT or Tax categories
+   */
   categorizeTransaction: Transaction,
-  /** Update a transaction with a category (with an optional custom booking date for VAT or Tax categories) and/or a personal note */
+  /** Categorize a transaction with an optional custom booking date for VAT or Tax categories, and add a personal note */
```

Comment:
Please keep the first comment line about `update`:
`  /** Update a transaction with a category (with an optional custom booking date for VAT or Tax categories) and/or a personal note */
`